packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -635,57 +635,57 @@ export class MgtPersonCard extends MgtTemplatedComponent {
     let email: TemplateResult;
     if (getEmailFromGraphEntity(person)) {
       email = html`
-         <div class="icon"
-           @click=${() => this.emailUser()}
-           @mouseenter=${() => this.setHoveredState('email', true)}
-           @mouseleave=${() => this.setHoveredState('email', false)}
-           tabindex=0
-           role="button">
-           ${this.isHovered('email') ? getSvg(SvgIcon.SmallEmailHovered) : getSvg(SvgIcon.SmallEmail)}
-         </div>
-       `;
+        <fluent-button class="icon"
+          @click=${() => this.emailUser()}
+          @mouseenter=${() => this.setHoveredState('email', true)}
+          @mouseleave=${() => this.setHoveredState('email', false)}>
+          <span class=${classMap({ filled: this.isHovered('email'), regular: !this.isHovered('email') })}>
```

Comment:
I don't think we need this span anymore, we can also delete the isHovered method and the setHoveredState event handlers for all the buttons